### PR TITLE
Render the @return annotation

### LINKED ISSUE #30: @return annotations don't generate any docs
_From @enguyen on October 17, 2014 19:16_

See, e.g. the @return statement in sayHello's comment block -- no corresponding text in the generated doc.

In general, it would be great to have clearer documentation around which JSDoc annotations are supported, and if there are any restrictions (e.g. @param annotations may not be multiple lines.)

_Copied from original issue: PolymerLabs/seed-element#36_

### PR DESCRIPTION
**Pull request by ulybu**
This fixes #30 .

Works in combination with https://github.com/Polymer/context-free-parser/pull/7

Here's a demo http://ulybu.github.io/core-doc-viewer/#core-icon (look at the `methods` section)

This PR also underlines some pragma detail sections like the `Events details`, `Method parameters` or the new `Returns`

(I got a company-CLA signed, Hewlett-Packard)

### CODE REVIEW

**Comment by ulybu** [comment]
@ebidel Are there additional action needed from me ?

**Comment by ebidel** [comment]
Did you sign the CLA?

File: elements/core-doc-page.css [hunk 1]
```diff
@@ -51,6 +51,9 @@ h3 {
 .box:not(.top) .details .params {
   margin-top: 40px;
 }
+.params>p:first-child{
```
Comment (ebidel) [inline elements/core-doc-page.css]: spacing : `params > p:first-child {`

**Comment by ulybu** [comment]
Hi, friendly ping

File: elements/core-doc-page.html [hunk 2]
```diff
@@ -168,6 +168,13 @@ <h3 id="{{data.name}}.methods">Methods</h3>
                       </template>
                     </div>
                   </template>
+                  <template if="{{method.return}}">
+                    <div class="params">
+                      <p>Returns:</p>
+                        <p><code>&lt;<em>{{method.return.type}}</em>&gt;</code></p>
```
Comment (ebidel) [inline elements/core-doc-page.html]: Can you fix the indentation?

File: elements/core-doc-page.css [hunk 3]
```diff
@@ -51,6 +51,9 @@ h3 {
 .box:not(.top) .details .params {
   margin-top: 40px;
 }
+.params > p:first-child{
```
Comment (ebidel) [inline elements/core-doc-page.css]: child{ -> child {

**Comment by ebidel** [comment]
Sorry just more nits on spacing.

**Comment by ulybu** [comment]
No problem!

**Comment by ulybu** [comment]
(done)

**Comment by ebidel** [comment]
:+1:

**Pull request merged**